### PR TITLE
Fix Map Card styling boundaries

### LINKED ISSUE #13681: Map card shows blank if either panel is true or if in a vertical-stack
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

The map card sometimes shows as empty, with just the "crosshairs" icon visible and the rest blank. 
See the picture below.

image

### Describe the behavior you expected

Map card should show correctly.

### Steps to reproduce the issue

The extract below causes the map card to show blank. It can be made to work if: (a) `panel` is set to `false`; OR (b) the `vertical-stack` is removed; OR (c) I enter the "Edit Raw YAML" and exit back to the dashboard; OR (d) I change the YAML and use the 'soft refresh' that appears in the bottom left to reload the page. 

Note that (c) and (d) here are only temporary, and a hard "F5" refresh causes the card to disappear again -- only (a) and (b) above are permanent fixes (albeit insufficient ones, given I need the card to appear in a panel and in a vertical stack).

```
views:
  - title: Home
    panel: true
    cards:
      - type: vertical-stack
        cards:
          - type: map
            entities:
              - entity: device_tracker.blue_jay_location_tracker
```

### What version of Home Assistant Core has the issue?

2022.9.1

### What was the last working version of Home Assistant Core?

_No response_

### In which browser are you experiencing the issue with?

Google Chrome 104.0.5112.102 (Windows) + other browsers on Linux and Raspbian)

### Which operating system are you using to run this browser?

Windows (plus Linux and Raspbian)

### State of relevant entities

_No response_

### Problem-relevant frontend configuration

_No response_

### Javascript errors shown in your browser console/inspector

```txt
None. Here's the console log - nothing shows as awry.

BROWSER_MOD 2.1.3 IS INSTALLED    
card-mod.js:5 CARD-MOD 3.1.5 IS INSTALLED
kiosk-mode.js?hacstag=497319128173:64 ≡ kiosk-mode            ⋮ 
version 1.7.3            
vacuum-card.js:4  VACUUM-CARD  2.6.2
```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jrobles98**
## Proposed change

Fixed Bug for Map Card that becomes blank when inside Horizontal/Vertical stack. Also happens when it is inside the Panel or Sidebar kind of Dashboards.

This fix solves the problem presented in [this issue](https://github.com/home-assistant/frontend/issues/13681)

## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #13681 

## Checklist

- [x] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi jrobles98

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant/frontend#14050).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: src/panels/lovelace/cards/hui-map-card.ts [hunk 1]
```diff
@@ -375,6 +375,7 @@ class HuiMapCard extends LitElement implements LovelaceCard {
 
       #root {
         position: relative;
+        height: 100%;
       }
 
       :host([ispanel]) #root {
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-map-card.ts:381]: Remove this rule as it is not needed anymore now.

**Review by bramkragten — DISMISSED** [review]

File: src/panels/lovelace/cards/hui-map-card.ts (hunk 1 above)
Comment (jrobles98) [reply src/panels/lovelace/cards/hui-map-card.ts:381]: But that is actually the fix itself for the map problem that I am trying to solve.
Or do you mean the whole `#root` rule?

Comment (bramkragten) [reply src/panels/lovelace/cards/hui-map-card.ts:381]: I mean the `:host([ispanel]) #root` rule, that is duplicate now as it is already in the `#root` rule

**Review by bramkragten — APPROVED** [review]

**Pull request merged**